packages/grid/stories/grid.doc.stories.mdx

```diff
@@ -148,24 +169,60 @@ This grid also shows an example of a custom cell displaying a basic chart. Switc
 
 ### Editable cells
 
-Grid allows you to make cell values editable in place.
-Use the `CellEditor` component as a child of `GridColumn` to make the column editable.
-`CellEditor` is a non-rendered component.
-`CellEditor` is expected to have one child representing the specific editor component to use for the column, e.g. `TextCellEditor`, `DropdownCellEditor` or `NumericCellEditor`.
+The grid allows you to make cell values editable in place.
+
+Use the `CellEditor` (a non-rendered) component as a child of `GridColumn` to make the column editable.
+`CellEditor` is expected to have one child representing the specific editor component to use for the column.
 The child of the `CellEditor` component gets rendered in the focused cell when the grid goes to edit mode.
 
+We have [three sample components](https://uitk.pages.dev/?path=/story/grid-new-grid--editable-cells) created for use as `CellEditor`'s child,
+i.e. `TextCellEditor`, `DropdownCellEditor` and `NumericCellEditor`.
+You can easily create any components tailored to your needs using the same approach.
+
 <Canvas>
   <Story id="grid-new-grid--editable-cells" decorators={[withFlexGap]} />
 </Canvas>
 
+To trigger edit mode on cells on your...
+
+- keyboard: `backspace` key _or_ `F2` key _or_ simply start typing
+- mouse: double click
+
 ### Pagination
 
-Implement pagination using the `Paginator` component. It is not a feature of the grid itself.
+[Implement pagination](https://uitk.pages.dev/?path=/story/lab-pagination--default-pagination) using the `Paginator` component. It is not a feature of the grid itself.
+
+```
+import { Pagination, Paginator } from "@jpmorganchase/uitk-lab";
+
+interface PaginationProps {
+  count: number
+  page?: number
+  onPageChange?: ((page: number) => void)
+  initialPage?: number
+  compact?: Boolean
+}
+
+interface PaginatorProps {
+  boundaryCount?: number;
+  siblingCount?: number;
+  showPreviousNext?: boolean;
+  FormFieldProps?: Partial<FormFieldProps>;
+}
+```
 
 <Canvas>
   <Story id="grid-new-grid--grid-pagination" decorators={[withFlexGap]} />
 </Canvas>
 
+### Writing tests for Grid component
+
+The grid utilizes virtualization to load content and renders only what’s in the viewport.
+As previously fetched data leave the viewport, so do the DOM notes.
```

Comment:
```suggestion
As previously fetched data leave the viewport, so do the DOM nodes.
```

Reply:
Also, not sure I understand this sentence. What do you mean by "previously fetched data leave the viewport"?

Reply:
Does "previously fetched data leave the viewport (data scrolled out of view in grid)" help provide better context?

Reply:
gotcha, all done!